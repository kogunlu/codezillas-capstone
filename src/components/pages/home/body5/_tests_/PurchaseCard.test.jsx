import React from "react";
import renderer from "react-test-renderer";
import PurchaseCard from "../PurchaseCard";


it('Renders correctly for About Component', () => {
    const tree = renderer
    .create(<PurchaseCard />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
