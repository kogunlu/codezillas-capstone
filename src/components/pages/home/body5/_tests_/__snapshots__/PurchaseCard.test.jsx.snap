// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for About Component 1`] = `
<div
  className="border border-solid w-72 lg:w-96 border-gray-200 rounded-lg shadow-xl"
>
  <div
    className="flex flex-col m-6"
  >
    <h1
      className="text-3xl font-semibold pt-12  flex flex-col items-center justify-center 2xl:text-5xl"
    />
    <h2
      className="text-gray-400 text-xl pt-3 font-medium flex flex-col items-center justify-center 2xl:text-3xl 2xl:py-8"
    />
    <h2
      className="text-base pt-2 font-normal 2xl:text-2xl pb-6"
    />
    <div
      className="flex flex-col items-center justify-center pb-10 2xl:text-3xl 2xl:pb-12"
    >
      <input
        className="py-2 px-7 hover:bg-cyan-200 bg-cyan-400 font-medium rounded"
        type="button"
      />
    </div>
  </div>
</div>
`;
